ui/components/ui/hardware-wallet-state/index.scss

```diff
@@ -0,0 +1,5 @@
+.hardware-wallet-state {
+  &__error {
+    margin: 16px 0;
+  }
+}
```

Comment:
Not sure this file is needed anymore?

Reply:
I went back and forth a few times of needing and not needing it - removed

Reply:
Oh I wasn't aware it was used. I couldn't find it as a className anywhere? Is it possibly left over from when the old component was used?

<img width="405" alt="Screenshot 2023-02-22 at 5 52 31 PM" src="https://user-images.githubusercontent.com/8112138/220797970-975fbc49-d788-4cb1-af92-6131c5ef21f6.png">

Reply:
Sorry, I have removed it - it wasn't used (anymore)